Municipality/azc.mod2g

```diff
@@ -1,13 +1,52 @@
-use dummy as knowledge.
-
-order = linearall.
-module azc{
-	%Check if there is still need for an azc (id 650)
-	if bel(building(BuildingID,_,OwnerID,_,_,650,_)) then exit-module.
-	%Check if we have budget for an azc
-	if bel(money(MoneyAmount),MoneyAmount<100000) then exit-module.
-	%Check if there is space for an azc
-
-	%Adopt if all preconditions are met
-	if true then adopt(buildAZC).
+use dummy as knowledge.
+
+order = random.

```

Comment:
otherwise it will only build small azc's, now it will randomly chose between large and small